doc/item_group.txt

```diff
@@ -40,6 +40,8 @@ The table below explains which fields are accessed in each.
 
 GroupID: See the "Item Group ID" section in 'src/map/itemdb.h'.
          Supports IG_* constants.
+	and "Item Group ID" section in 'src/map/script_constants.h'.
+	 Supports export_constant(IG_* constants.
```

Comment:
```
         See the "item groups" section in 'src/map/script_constants.h".
         Supports export_constant(IG_*) constants.
```

Reply:
do you mean remove the first line ? (See the "Item Group ID" section in 'src/map/itemdb.h'.)
if not i think for the second line it's better to start it with (and) so they know that there is two files must be edited to add new IG